client/state/billing-transactions/actions.js

```diff
@@ -33,3 +36,26 @@ export const requestBillingTransactions = () => {
 			} );
 	};
 };
+
+export const sendBillingReceiptEmail = ( receiptId ) => {
```

Comment:
Just an fyi to keep 👀  out for the new data-layer+http layer in Calypso. A thunk here is fine for now, since the pattern isn't quite fleshed out yet.